hnn_core/network.py

```diff
@@ -1340,6 +1386,10 @@ def plot_cells(self, ax=None, show=True):
         """
         return plot_cells(net=self, ax=ax, show=show)
 
+    @copy_doc(write_network)
+    def write(self, fname, overwrite=True, save_unsimulated=False):
```

Comment:
also should `overwrite` be `False` or `True` by default?

Reply:
Should we give parameters `write_cell_response` and `write_extracellular_array` separately. `write_output` might be better because some simulation outputs are linked to `external_drives` also.

Reply:
`external_drives` are definitely not "outputs" ... I see now why it was called "unsimulated". But we should do the following:

when writing: `save_output=False` will save network and external drives but not extracellular array and cell response.

when reading, two different parameters: `read_drives=True` will also read the external drives and `read_output=True` will also read the extracellular array and cell response

Reply:
If `read_drives=False` we won't be attaching drives at all?